.mockery.yaml

```diff
@@ -1,11 +1,11 @@
 with-expecter: true
-dir: "{{.InterfaceDir}}"
+dir: "{{.InterfaceDir}}/testutils"
 mockname: "{{.InterfaceName}}Mock"
-outpkg: "{{.PackageName}}_test"
-filename: "mock_{{.InterfaceName}}_test.go"
+outpkg: "testutils"
+filename: "mock_{{.InterfaceName}}.go"
 inpackage: true
 issue-845-fix: true
 packages:
-  github.com/kyma-project/lifecycle-manager/pkg/gatewaysecret:
+  github.com/kyma-project/lifecycle-manager/internal/gatewaysecret:
     interfaces:
       Client:
```

Comment:
Adapted this config to produce the MockClient at the location we had it in already (this was out of sync). We also need to consider aligning our expectations. Right now, the mock client is not stripped as part of testing code, it should be though.

Reply:
https://github.com/kyma-project/lifecycle-manager/issues/2422